core/src/commonMain/kotlin/com/ustadmobile/core/impl/appstate/AppUiState.kt

```diff
@@ -24,7 +24,7 @@ data class AppUiState(
     val loadingState: LoadingUiState = LoadingUiState(),
     val title: String? = null,
     val navigationVisible: Boolean = true,
-    val hideBottomNavigation: Boolean = false,
+    var hideBottomNavigation: Boolean = false,
```

Comment:
This should not be changed. AppUiState should be immutable in order to avoid side effects.